package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "ng-high-memory": "node --max_old_space_size=8000 ./node_modules/@angular/cli/bin/ng",
     "ng": "npx ng",
-    "start": "npx ng serve --serve-path=/ids-enterprise-ng-demo",
```

Comment:
--serve-path in the scripts is no longer needed now that this is defined in angular.json